### DISCUSSION TITLE
Twitter error response: status code = 403 when fetching favorites

### DISCUSSION THREAD

**Discussion by NTHieu95** [tweepy/tweepy#2133] [Questions] [answered]
I've been using tweepy for a while but I didn't care much about how the twitter APIs have changed. And recently I found out that my app has been suspended for "violating twitter rules" (I'm using a script that sometimes fetch all of my favorite tweets using `tweepy.Cursor(api.favorites, id=User, tweet_mode="extended").items(4000)` ).
So I made a new app and tried to make thing work again but I keep getting `Twitter error response: status code = 403` exception.

I'm using free plan and apparently according to this link:
https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#item0
I'm understanding that we cannot pull tweets when using free plan anymore. Am I understanding it correctly or am I missing something?

**Comment by Harmon758** [comment, marked as answer]
Yes, retrieving Tweets is not available with Free access.
https://twittercommunity.com/t/announcing-new-access-tiers-for-the-twitter-api/188728
https://twittercommunity.com/t/deprecation-of-legacy-access-tiers-in-the-twitter-api/196162
https://developer.twitter.com/en
